src/main/java/mesosphere/marathon/metrics/Metrics.scala

```diff
@@ -64,6 +68,11 @@ class Metrics @Inject() (val registry: MetricRegistry) {
 }
 
 object Metrics {
+  class Counter(counter: com.codahale.metrics.Counter) {
```

Comment:
Actually, this is great. We missed the `Counter` type before and used stuff like the `AtomicIntGauge` instead. I am just not sure whether it's a good idea to introduce this in this patch.

Either way, it probably makes sense to create an issue and change `master` to using counters instead of `AtomicIntGauge`?

Also, please document that it is a gauge, since counter is not a metric category in statsd, I wouldn't know where to find it otherwise.